source/src/molecular-unfolding/cdk/utils/custom-resource-lambda/create-event-rule/cloudformation-custom-resource.d.ts

```diff
@@ -0,0 +1,68 @@
+import { Handler } from "./handler";
```

Comment:
Again, don't copy this type from other packages or define them by yourself. There is a well-known type package for `aws-lambda`.

Reply:
changed to use "@types/aws-lambda"